.github/docker-soak-tests/docker-compose.yml

```diff
@@ -0,0 +1,133 @@
+version: "3.7"
+services:
+  otel:
+    image: otel/opentelemetry-collector-contrib:latest
+    command: --config /otel-collector/collector-config.yml --log-level debug
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+      - HOSTMETRICS_COLLECTION_INTERVAL_IN_SECONDS
+      - LOG_GROUP_NAME
+      - LOGS_NAMESPACE
+      - LOG_STREAM_NAME
+      - PROCESS_COMMAND_LINE
+      - PROCESS_EXECUTABLE_NAME
+    volumes:
+      - ./otel-collector:/otel-collector
+      - type: bind
+        source: /proc
+        target: /proc
+    ports:
+      - '4317:4317'
+    user: "${UID}:${GID}"
+
+  app:
+    image: ${APP_IMAGE}
+    environment:
+      - INSTANCE_ID
+      - LISTEN_ADDRESS
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+      - AWS_DEFAULT_REGION=us-west-2
+      - SAMPLE_APP_LOG_LEVEL=ERROR
+      - OTEL_RESOURCE_ATTRIBUTES=service.name=aws-otel-integ-test
+      - OTEL_EXPORTER_OTLP_ENDPOINT=grpc://otel:4317
+    ports:
+      - '8080:8080'
+
+  load-generator:
+    build:
+      context: ./load-generator
+    environment:
+      - SOAK_TEST_DURATION_IN_MINUTES
+    depends_on:
+      - otel
+      - app
+
+  alarms-poller:
+    build:
+      context: ./alarms-poller
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+      - AWS_DEFAULT_REGION=us-west-2
+      - HOSTMETRICS_COLLECTION_INTERVAL_IN_SECONDS
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - load-generator
+
+# # To run on Cloud Desktop:
```

Comment:
Better to delete this from the PR (it's referencing something from your personal repo anyways)

Reply:
Sounds good, I have the parameters in my personal repo if we want to know how to run it locally in the future.